code/game/gamemodes/cult/cult.dm

```diff
@@ -109,6 +108,7 @@
 					possible_targets += player.mind
 		if(possible_targets.len > 0)
 			sacrifice_target = pick(possible_targets)
+			GLOB.sac_target = sacrifice_target
```

Comment:
I missed the tutorial on how to make subsystem tickers function like global variables